geoh5py/ui_json/ui_json.py

```diff
@@ -47,3 +64,22 @@ def get_data(param: Parameter | FormParameter):
             out[param.name] = get_data(param)
 
         return out
+
+    def validate(self):
+        """Validates uijson data against a pool of enforcers."""
+
+        uijson = self.to_dict()
+        self.enforcers.enforce(uijson)
+
+    @property
+    def name(self) -> str:
+        """Returns a name for the uijson file."""
+
+        uijson = self.to_dict()
+        name = "uijson"
+        if "title" in uijson:
+            name = uijson["title"]
+        elif "geoh5" in uijson:

```

Comment:
yeah, I think you're right.. I'll change it.

Reply:
I'll make the change in the next issue since the parameter value access has changed.  Since geoh5 is a required parameter, I'll just hardwire the name to the h5file.stem, no logic.